src/gentropy/finngen_finemapping_ingestion.py

```diff
@@ -31,13 +31,17 @@ def __init__(
         """
         # Read finemapping outputs from the input paths.
 
-        finngen_finemapping_df = FinnGenFinemapping.from_finngen_susie_finemapping(
-            spark=session.spark,
-            finngen_susie_finemapping_snp_files=finngen_susie_finemapping_snp_files,
-            finngen_susie_finemapping_cs_summary_files=finngen_susie_finemapping_cs_summary_files,
-        )
-
-        # Write the output.
-        finngen_finemapping_df.df.write.mode(session.write_mode).parquet(
-            finngen_finemapping_out
+        (
+            FinnGenFinemapping.from_finngen_susie_finemapping(
+                spark=session.spark,
+                finngen_susie_finemapping_snp_files=finngen_susie_finemapping_snp_files,
+                finngen_susie_finemapping_cs_summary_files=finngen_susie_finemapping_cs_summary_files,
+            )
+            # Flagging sub-significnat loci:
+            .validate_lead_pvalue(
+                pvalue_cutoff=FinngenFinemappingConfig().finngen_finemapping_lead_pvalue_threshold
```

Comment:
Same comment: `FinngenFinemappingConfig().finngen_finemapping_lead_pvalue_threshold` has to be an init parameter of the step.